package.json

```diff
@@ -1,6 +1,8 @@
 {
```

Comment:
As a note on this, we should add a LICENSE (we can copy https://github.com/aragonone/voting-connectors/blob/master/LICENSE)